src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6352,6 +6352,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_NonPrivateAPIInRecord" xml:space="preserve">
     <value>Record member '{0}' must be private.</value>
   </data>
+  <data name="WRN_PrecedenceInversion" xml:space="preserve">
+    <value>Operator '{0}' cannot be used here due to precedence. Use parentheses to disambiguate.</value>
```

Comment:
>Operator '{0}' cannot be used here due to precedence [](start = 11, length = 52)

This is a warning so it seems strange to say "Operator cannot be used here due to precedence." We're allowing the operator so presumably it can be used. Should the message be more specific? Perhaps "Operator cannot be used here with the expected precedence."

Reply:
How would "usual" add any clarity?  We're accepting the operator because we're not paying any attention to precedence at all.

---
In reply to: [463301820](https://github.com/dotnet/roslyn/pull/46239#discussion_r463301820) [](ancestors = 463301820)